devices/schneider_electric.js

```diff
@@ -152,6 +152,18 @@ module.exports = [
             await reporting.onOff(endpoint);
         },
     },
+    {
+        zigbeeModel: ['NHPB/SWITCH/1'],
+        model: 'NHPB/SWITCH/1',
```

Comment:
Please add the model as written on the device, I expect something like `U201SRY2KWZB` (since other Schneider devices uses such model).